testing/bdd/features/validation/smpte_constraints.feature

```diff
@@ -3,22 +3,48 @@ Feature: SMPTE-related attribute constraints
     Given an xml file <xml_file>
     And it has frameRate <frame_rate>
     And it has timeBase <time_base>
+    And it has frameRateMultiplier <frame_rate_multiplier>
+    And it has dropMode <drop_mode>
+    And it has markerMode <marker_mode>
     Then document is valid
 
     Examples:
-    | xml_file   | frame_rate | time_base |
-    | smpte.xml  | 25         | smpte     |
-    | smpte.xml  | 20         | smpte     |
+    | xml_file   | frame_rate | time_base | frame_rate_multiplier | drop_mode | marker_mode   |
+    | smpte.xml  | 25         | smpte     | 1 1                   | nonDrop   | continuous    |
+    | smpte.xml  | 20         | smpte     | 1 1                   | nonDrop   | discontinuous |
+    | smpte.xml  | 30         | smpte     | 1000 1001             | dropNTSC  | continuous    |
+    | smpte.xml  | 30         | smpte     | 1000 1001             | dropNTSC  | discontinuous |
+    | smpte.xml  | 30         | smpte     | 1000 1001             | dropPAL   | continuous    |
+    | smpte.xml  | 30         | smpte     | 1000 1001             | dropPAL   | discontinuous |
 
   # These tests are not all passing because the missing semantic validation piece
   @skip
   Scenario: Invalid SMPTE head attributes
     Given an xml file <xml_file>
     And it has frameRate <frame_rate>
     And it has timeBase <time_base>
+    And it has frameRateMultiplier <frame_rate_multiplier>
+    And it has dropMode <drop_mode>
+    And it has markerMode <marker_mode>
     Then document is invalid
 
     Examples:
-    | xml_file   | frame_rate | time_base |
-    | smpte.xml  |            | smpte     |
-    | smpte.xml  | 25         | clock     |
+    | xml_file   | frame_rate | time_base | frame_rate_multiplier | drop_mode | marker_mode   |
+    | smpte.xml  |            | smpte     |                       |           |               |
+    | smpte.xml  | 25         | smpte     | 1 1                   | dropPAL   | continuous    |
+    | smpte.xml  | 30         | smpte     | 10001001              | dropPAL   | continuous    |
```

Comment:
No no that was my misreading - absolutely keep this as an example of invalid. Sorry!